DataFormats/HLTReco/src/classes_def.xml

```diff
@@ -58,17 +54,13 @@
   <class name="std::vector<trigger::TriggerEvent::TriggerFilterObject>"/>
   <class name="trigger::TriggerEvent" ClassVersion="11">
    <version ClassVersion="11" checksum="3351458717"/>
-   <version ClassVersion="10" checksum="1112210423"/>
   </class>
   <class name="trigger::TriggerEventWithRefs::TriggerFilterObject" ClassVersion="13">
    <version ClassVersion="13" checksum="1672519577"/>
-   <version ClassVersion="12" checksum="2301242282"/>
   </class>
   <class name="std::vector<trigger::TriggerEventWithRefs::TriggerFilterObject>"/>
-  <class name="trigger::TriggerEventWithRefs" ClassVersion="12">
-   <version ClassVersion="12" checksum="3347721344"/>
-   <version ClassVersion="11" checksum="2025606788"/>
-   <version ClassVersion="10" checksum="4075310205"/>
+  <class name="trigger::TriggerEventWithRefs" ClassVersion="19">
+   <version ClassVersion="19" checksum="1258968436"/>
   </class>
 
   <class name="edm::Wrapper<trigger::TriggerObjectCollection>" splitLevel="0"/>
```

Comment:
Please restore all class versions which were in the file /CMSSW 80X / before your editing.